### PR TITLE
Remove MigrationUtil

### PR DESCRIPTION
**Pull request by minwoox**
Motivation:
I found out `MigrationUtil` is not used anymore that is intruced token migration in #133.

Modifications:
- Remove `MigrationUtil`
- Move initialization logic to `ProjectInitializer`
- Clean up some logs in `ZooKeeperCommandExecutor`

Result:
- Less code

### CODE REVIEW

File: server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java [hunk 1]
```diff
@@ -725,7 +725,7 @@ private synchronized void replayLogs(long targetRevision) {
 
                     if (!Objects.equals(expectedResult, actualResult)) {
                         throw new ReplicationException(
-                                "mismatching replay result at revision " + info.lastReplayedRevision +
+                                "mismatching replay result at revision " + nextRevision +
```
Comment (ikhoon) [inline server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java:728]: 👍

File: server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java [hunk 2]
```diff
@@ -736,22 +736,22 @@ private synchronized void replayLogs(long targetRevision) {
                     // same replicaId. skip
                 }
 
-                info.lastReplayedRevision = nextRevision;
                 updateLastReplayedRevision(nextRevision);
+                info.lastReplayedRevision = nextRevision;
                 if (nextRevision == targetRevision) {
                     break;
+                } else {
+                    nextRevision = info.lastReplayedRevision + 1;
```
Comment (ikhoon) [inline server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java:744]: nit:
```suggestion
                    nextRevision++;
```

**Review by ikhoon — APPROVED** [review]
Nice work!

File: server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java (hunk 2 above)
Comment (minwoox) [reply server/src/main/java/com/linecorp/centraldogma/server/internal/replication/ZooKeeperCommandExecutor.java]: Fixed. 😉

**Comment by minwoox** [comment]
Thanks for reviewing. 😉

**Pull request merged**